### ISSUE TITLE
False positive duplicate-argument-name with place-holder

### ISSUE THREAD

**Issue by SnarkBoojum** [PyCQA/pylint#3938]
The following function:
```
def ignore(_, _):
    pass
```
will trigger the duplicate-argument-name, while it shouldn't : it isn't a real argument.

**Comment by SnarkBoojum** [comment]
Sigh... it's an error in Python too, so I'm supposed to use *_ : invalid report, closing.

Sorry for the noise!
